.github/workflows/olm_pr.yaml

```diff
@@ -43,12 +43,11 @@ jobs:
       - name: Generate OLM bundle
         env:
           TOOL_VERSION: ${{ github.event.inputs.olmBundleToolVersion }}
-          PREVIOUS_VERSION: "v0.0.1" # remove me after the pr on community-operators repo is merged
```

Comment:
It is no longer relevant?

Reply:
the pr w/ 0.8.3 [got](https://github.com/k8s-operatorhub/community-operators/pull/356) merged so for the next release this value will be calculated as "the last but one release"